data/transit/route/bus.json

```
@@ -7879,6 +7879,18 @@
         "route": "bus"
       }
     },
+    {
+      "displayName": "Mistral",
+      "id": "reseaumistral-add5eb",
+      "locationSet": {"include": ["fx"]},
+      "matchNames": ["mistral"],
```

If “Réseau Mistral” is a mistypo, you can add it to matchNames so that other can find it and rename.

```suggestion
      "matchNames": ["mistral","Réseau Mistral"],
```